src/languages/es.ts

```diff
@@ -1952,6 +1952,10 @@ export default {
             status: 'Estado',
             enabled: 'Activada',
             disabled: 'Desactivada',
+            errors: {
```

Comment:
I think we should just remove this one and use `common.genericErrorMessage`

Reply:
Haha I'm not sure where it came from, but I think we can just reuse `common.genericErrorMessage` for now and update the copy later if we need to since this feature is still WIP